File: Python/(Beecrowd)1066 - Even, Odd, Positive and Negative.py
even = 0
odd = 0
posit = 0
negat = 0

for i in range(5):
    number = int(input())
    
    if number % 2 == 0:
        even += 1
    else:
        odd += 1

    if number > 0:
        posit += 1
    elif number < 0:
        negat += 1

print(even, "valor(es) par(es)")
print(odd, "valor(es) impar(es)")
print(posit, "valor(es) positivo(s)")
print(negat, "valor(es) negativo(s)")

File: C/(Beecrowd)1066 - Even, Odd, Positive and Negative.c
#include <stdio.h>

int main() {
    int n, posit = 0, negat = 0, even = 0, odd = 0;

    for (int i = 0; i < 5; i++) {
        scanf("%d", &n);

        if (n > 0) {
            posit ++;
        }
        if (n < 0) {
            negat ++;
        }

        if (n % 2 == 0) {
            even ++;
        } else {
            odd ++;
        }
    }

    printf("%d valor(es) par(es)\n", even);
    printf("%d valor(es) impar(es)\n", odd);
    printf("%d valor(es) positivo(s)\n", posit);
    printf("%d valor(es) negativo(s)\n", negat);

    return 0;
}